Automattic/wp-api-console
# Improve the WordPress developer console with a few new features.

While developing some new endpoints in WordPress.com I've realised that I needed to compare data returned from those endpoint and compare them (I was refactoring and needed to be completely sure that the endpoint was returning the same exact amount of information)

With this Pull Request I also wanted to make use of functional component, but seems that the react version we are using does not support it. So I preferred to make the change smaller and not update React at all.

This Pull Request tries to address some of the difficulties I found during that development with the WordPress developer console.

- Now it is possible to copy the JSON raw response directly with a button. Before I was opening the browser console and inspecting the network call, plus this network call had the HTML envelope and was also being a bit disturbing 
- See the response in RAW format, without the tree view.
- This PR also refactors the components a bit so they make more sense.

![image](https://user-images.githubusercontent.com/5689927/155840530-1145ff69-fb6a-4bdf-8f46-9d09f545d0d2.png)
![image](https://user-images.githubusercontent.com/5689927/155840567-888e3e34-8ee5-49a8-a20b-f93f1a2fad86.png)

### Testing instructions

Check that the default view for every call is still the Tree View. I don't want to disturb others that are used to this view.
Check that you can copy the results with the scissors button at the end of the row. You should be able to copy the contents regardless of the selected view
Check that regular calls are still working to any other endpoint you were using before (WP .COM & WP REST)
